Venmo/src/main/java/com/braintreepayments/api/VenmoRequest.java

```diff
@@ -120,8 +269,16 @@ public int describeContents() {
     @Override
     public void writeToParcel(Parcel parcel, int i) {
         parcel.writeByte((byte) (shouldVault ? 1 : 0));
+        parcel.writeByte((byte) (collectCustomerBillingAddress ? 1 : 0));
+        parcel.writeByte((byte) (collectCustomerShippingAddress ? 1 : 0));
         parcel.writeString(profileId);
         parcel.writeString(displayName);
         parcel.writeInt(paymentMethodUsage);
+        parcel.writeString(subTotalAmount);
+        parcel.writeString(discountAmount);
+        parcel.writeString(taxAmount);
+        parcel.writeString(shippingAmount);
+        parcel.writeString(totalAmount);
+        parcel.writeTypedList(lineItems);
```

Comment:
```suggestion
        parcel.writeString(shippingAmount);
        parcel.writeString(taxAmount);
        parcel.writeString(totalAmount);
        parcel.writeTypedList(lineItems);
```
This needs to be in the same order as the Parcel constructor.